engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java

```diff
@@ -806,10 +806,6 @@ public State emptyState() {
     return State.create(this);
   }
 
-  public int getMaxUnboxingLayouts() {
-    return 10;
```

Comment:
Removal of this method fixes [SuggestionBuilderTest](https://github.com/enso-org/enso/actions/runs/7510257625/job/20448449518#step:10:3256). Done in 580c3d80984e0178e43dd1e41ea580c682fa505a